src/CreatePhotosSlideshow/CreatePhotosSlideshow.tsx

```diff
@@ -96,41 +127,45 @@ const CreatePhotosSlideshow = (): JSX.Element => {
       setProgress(parseFloat((ratio * 100).toFixed(1)));
     });
 
-    for (const index of orderedItems) {
-      ffmpeg.FS(
-        'writeFile',
-        getCleanName(files[index]),
-        new Uint8Array(correctedFileArrayBuffer[index])
-      );
-    }
+    const inputPaths: Array<string> = [];
+    orderedItems.forEach(async (itemPosition, index) => {
+      const fileName = getCleanName(files[itemPosition].name);
+      inputPaths.push(`file ${fileName}\nduration 2`);
+      ffmpeg.FS('writeFile', fileName, await fetchFile(files[itemPosition]));
```

Comment:
await might not behave as one expects it inside forEach instead of a normal for loop. The loop is likely to continue working on the next items, but that might be an ok behavior in this

Reply:
Make sense I'll switch to for .. of